experiments/modern/src/maki-interpreter/virtualMachine.js

```diff
@@ -82,8 +149,9 @@ async function interpret(start, program, { logger = null }) {
       case 48: {
         const a = stack.pop();
         const b = stack.pop();
-        b.setValue(a);
```

Comment:
Without this some of the tests were getting variables that contained variables as their value. I think whenever we set a variables value it should always be the concrete value.

Reply:
Hmm. This is fine for now, but I suspect/hope we can remove all this wrapping/unwrapping at some point.

Reply:
Yea, agreed. I actually tried to remove all the wrapping and ensure that we only ever had literal values on the stack, so we could get rid of all the `instanceof` checks, but I wasn't able to get it to work. Hoping that we can figure out the "right" way, and figure with more tests running through the system we can be more confident in tweaks we make.